client/my-sites/activity/activity-log-item/activity-actor.jsx

```diff
@@ -27,6 +27,15 @@ const HAPPINESS_ACTOR = (
 	</div>
 );
 
+const SERVER_ACTOR = (
+	<div className="activity-log-item__actor">
+		<div class="server-actor-icon"></div>
+		<div className="activity-log-item__actor-info">
+			<div className="activity-log-item__actor-name">Server</div>
```

Comment:
Needs translation function

Reply:
I was originally thinking 'Server' should be hard-coded similarly to 'Jetpack' and 'Happiness Engineer'. I suppose it makes sense to translate, though ce21a232ec891e8ace2cd3148f3db9fe1b8b1c61

Reply:
Jetpack is a brand thus not translated but "Happiness Engineer" should probably also be translated. Seems just an oversight.